packages/theme-material-ui/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@hig/theme-material-ui",
+  "version": "1.0.0-alpha",
+  "description": "HIG Theme Material UI",
+  "author": "Autodesk Inc.",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Autodesk/hig.git"
+  },
+  "files": [
+    "build/*"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test": "hig-scripts-test",
+    "build": "yarn run build:themes",
+    "build:core": "hig-scripts-build",
+    "build:themes": "babel-node scripts/buildThemes",
+    "lint": "hig-scripts-lint",
+    "release": "hig-scripts-release"
+  },
+  "peerDependencies": {
+    "@hig/theme-context": "^3.0.0",
+    "@hig/theme-data": "^2.4.0"
+  },
+  "devDependencies": {
+    "@hig/babel-preset": "^0.1.1",
+    "@hig/eslint-config": "^0.1.0",
+    "@hig/jest-preset": "^0.1.0",
+    "@hig/scripts": "^0.1.2",
+    "mkdirp": "^0.5.1",
+    "dedent": "^0.7.0",
+    "@hig/semantic-release-config": "^0.1.0",
```

Comment:
a bit of a nitpick, but `@hig/semantic-release-config` should be moved up with the other `@hig` dev dependencies.